main.py

```diff
@@ -0,0 +1,84 @@
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.button import Button, ButtonBehavior
+from kivy.uix.image import Image
+from kivy.clock import Clock
+import random
+from kivy.core.audio import SoundLoader
+from kivy.app import App
+from kivy.uix.screenmanager import ScreenManager, Screen
+
+# Declaring screens for use in screen manager
+class BattleScreen(Screen):
+    def check_for_monster(self):
+        try:
+            return self.monster
+        except:
+            self.monster = self.random_stats()
+            return self.monster
+
+    def check_for_player(self):
+        try:
+            return self.player
+        except:
+            self.player = self.random_stats()
+            return self.player
+
+    def make_stat_text(self, title):                                        # title changes who's stats are displayed
+        if title == 'Monster:':
+            character = self.check_for_monster()
+        elif title == 'Player:':
+            character = self.check_for_player()
+        else:
+            character = self.random_stats()
+        for stat in range(0, len(character)):
+            title = title + '\n\n' + str(character[stat][0]) + ' = ' + str(character[stat][1])
+        return title
+
+    def random_stats(self):
+        hp = random.randint(14, 22)
+        strength = random.randint(1, 10)
+        if strength >= 5:
+            dexterity = random.randint(1, int(10 - (strength/4)))
+        else:
+            dexterity = random.randint(1, int(10 + (strength/4)))
+        will = random.randint(1, 10)
+        damageResistance = random.randint(0, 5)
+        character = []
```

Comment:
It may be more appropriate to use a dictionary here, here the keys are the names of the stats and the values are the values of the stats, rather than using a list of tuples.